pkgs/development/web/nodejs/v20.nix

```diff
@@ -16,5 +16,6 @@ buildNodejs {
     ./disable-darwin-v8-system-instrumentation-node19.patch
     ./bypass-darwin-xcrun-node16.patch
     ./node-npm-build-npm-package-logic.patch
+    ./common-gypi-cross.patch
```

Comment:
This patch is not nixpkgs-specific, right? Do you think you could make an upstream PR for this patch and `fetchpatch` it here?

I'm sure we wouldn't be the only ones to benefit from nodejs having working aarch64-linux -> x86_64-linux cross :)

Thank you for this work!